data/brands/shop/clothes.json

```diff
@@ -3306,7 +3306,7 @@
     {
       "displayName": "House",
       "id": "house-3937bd",
-      "locationSet": {"include": ["001"]},
+      "locationSet": {"include": ["pl", "ru", "ua", "cz", "ro", "sk", "ee", "bg", "hr", "lt", "lv", "hu", "rs", "kz", "fi", "si", "ba"]},
```

Comment:
A bit late, but do we want the locationSet list sorted to make it easier for others in future? I'm not sure if the CI currently checks for duplicates...

Or maybe the build command sorts it for us (or could do)?

Reply:
Yes I think the build command will sort it.

Reply:
Indeed it did:
https://github.com/osmlab/name-suggestion-index/commit/cb2dabf364d2d5ac1f3e7580cab68ec45b3bc1b2#diff-682a76e42ab2b82dbdfe83f676ce59a3cfa6a81751350a288d8c30eb1f49a4b0R3308